core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -368,7 +368,8 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
             displayName = request.name,
             spendProfile = Option(profile),
             billingProjectNamespace = request.namespace,
-            context
+            context,
+            buildPolicyInputs(request)
```

Comment:
I don't see that there's any test coverage for this method-- added a note in https://broadinstitute.slack.com/archives/C02L9L96YB1/p1714575548800209